src/logger/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+cmake_minimum_required(VERSION 3.16)
+
+set(LOGGER_SOURCES Logger.cpp)
+
+add_library(kp_logger ${LOGGER_SOURCES})
+target_link_libraries(kp_logger PUBLIC spdlog::spdlog)
```

Comment:
Important for spdlog to be optional, only required dependency should be fmt

Reply:
Done.